pkg/license/reporter.go

```diff
@@ -25,43 +25,44 @@ type ResourceReporter struct {
 }
 
 // NewResourceReporter returns a new ResourceReporter
-func NewResourceReporter(client client.Client) ResourceReporter {
+func NewResourceReporter(client client.Client, operatorNs string) ResourceReporter {
 	c := k8s.WrapClient(client)
 	return ResourceReporter{
 		aggregator: Aggregator{
 			client: c,
 		},
 		licensingResolver: LicensingResolver{
-			client: c,
+			client:     c,
+			operatorNs: operatorNs,
```

Comment:
This was always "" up until now, which lead to errors not that we actually to trial license checks.